Generate static libs with "_s" suffix

vivekarumugam opened the pull request:
In addition to that, added cleanobjdir as dependency for pxscene.so, pxwayland.so & pxwayland_s.a. 

This is to remove previously built object files before building either pxscene/pxwayland as they both have different compilation switches ("-DENABLE_RT_NODE")

mfiess commented:
I think the build scripts build librtCore.so and not lrtCore_s when building librtCore and libpxscene.so.  We should keep libpxscene.so using the .so version instead of the .a version for now.

vivekarumugam commented:
* Added librtRemote_s.a and librtCore_s.a as internal dependencies for libpxscene.so, libpxwayland.so and libpxwayland_s.a. In that way we need not to externally build those targets.

* Added separate rules to clean pxscene and pxwayland objects. This is done for the following reasons
  1) pxwayland and pxscene has different compilation parameters ("-DENABLE_RT_NODE")
  2) If we built pxwayland.so/pxwayland_s.a after building pxscene.so, previously built objects were taken for processing
  3) Cleaning entire obj/ directory leads to unnecessarily building rtCore_s.a again
